build/web-client-rpc.md

```diff
@@ -0,0 +1,55 @@
+# Web Client VS RPC Client
+
+If you're unsure which client to use for interacting with the Nimiq blockchain, this page offers a comparison between the Nimiq Web Client and the RPC so you can choose the tool that best fits your needs and expertise. {.subline}
+
+### TL;DR
+The Nimiq Web Client establishes direct connections with other peers in the network, making it a part of the blockchain network. In contrast, the RPC follows a client-server model, where clients connect to a single server that interacts with multiple peers on the network.
+
+<img class="object-contain max-h-[max(60vh,180px)]" src="/assets/images/protocol/network.png" alt="skip block struct" />
+
+## Nimiq Web Client
+
+The [Nimiq Web Client](/build/web-client/getting-started.md) is a JavaScript library that enables direct participation in the Nimiq PoS ecosystem directly from your web browser. This client is compiled from Rust to WebAssembly and offers a lightweight solution for users who want to interact with the Nimiq blockchain.
```

Comment:
The browser is one environment in where the Web Client can run, but not limited to. NodeJS is supported too I believe. 

@sisou how would you describe it as 'JavaScript runtimes' would be too inclusive as we haven't tested in Deno or Bun afaik.

Reply:
That's right. `wasm-pack` (which we use for building the client) also has support for Deno, which is on my list to add to the package.